scripts/exgdas_efsoi_update.sh

```diff
@@ -0,0 +1,413 @@
+#!/bin/ksh
```

Comment:
we now have 3 scripts;
exgdas_efsoi_update.sh, exgdas_efsoi.sh and the enkf_update script that are 90% same.

Reply:
I can see how the two update scripts can/should be combined, but exgdas_efsoi.sh, while based on the update script and using the same namelist file, does run a different executable with different input and output files.

Reply:
will reexamine